pom.xml

```diff
@@ -716,7 +718,7 @@
                         <log4j.configuration>file:${project.basedir}/../ksqldb-test-util/src/main/resources/log4j.properties</log4j.configuration>
                     </systemPropertyVariables>
                     <argLine>-verbose:gc -Xloggc:"${project.build.directory}/gc.log"</argLine>
-                    <argLine>-Duser.timezone=UTC -Xmx1g -Xms1g -XX:MaxPermSize=256m</argLine>
+                    <argLine>-Duser.timezone=UTC -Xmx1g -Xms1g</argLine>
```

Comment:
Why is this change needed?

Reply:
It was breaking otherwise. We were using java 11 before now it is `. set-cp-java-version` tool

Reply:
https://semaphore.ci.confluent.io/jobs/16f55e26-0406-4169-9370-aa66fc303b6b

Reply:
So in the later version( i see the job is using java 17), this option is not supported

Reply:
Yes, that's correct